.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
This file is for the editorconfig plugin, editor agnostic software that lets people format their code according to the style the jshint/eslint plugin expects.  Not for any particular editor, but works with any editor that has the right plugin.